packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -467,31 +1134,47 @@ const TableExample: React.FC = () => {
 
 #### Props
 
-#### Table
+##### Table
+
+All the valid HTML attributes for `table` are supported including the following props:
+
+###### `striped?: boolean`
+
+Sets a background color for even rows within the table.
 
-##### `tableLayout?: “auto” | “fixed”`
+###### `tableLayout?: "auto" | "fixed"`
 
 Sets the `table-layout` style of the Table. Defaults to `auto`.
 
-#### Tr Props
+###### `variant?: "default" | "borderless"`
 
-##### `verticalAlign?: “top” | “middle” | “bottom”`
+Sets the `border` style of the Table. Defaults to `default`.
+
+##### Tr Props
+
+All the valid HTML attributes for `tr` are supported including the following props:
+
+###### `verticalAlign?: "top" | "middle" | "bottom"`
 
 Sets the vertical alignment of the content within the Table row. Defaults to `middle`.
 
-#### Th Props
+##### Th Props
```

Comment:
do we want to include the prop here to set the Th scope to row?

Reply:
Thats part of the "All the valid HTML attributes for `th` are supported...", so I don't think it's necessary. We also call that out in the row headers section.